sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/models/IncludedPathTest.java

```diff
@@ -8,33 +8,25 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class IncludedPathTest {
 
     @Test(groups = {"unit"})
-    public void deserialize() {
-        String json = "{" +
-                "  'path': '\\/*'," +
-                "  'indexes': [" +
-                "    {" +
-                "      'kind': 'Range'," +
-                "      'dataType': 'String'," +
-                "      'precision': -1" +
-                "    }," +
-                "    {" +
-                "      'kind': 'Range'," +
-                "      'dataType': 'Number'," +
-                "      'precision': -1" +
-                "    }" +
-                "  ]" +
-                "}";
-        IncludedPath path = new IncludedPath(json);
-        List<Index> indexes = new ArrayList<>(path.getIndexes());
-        Assertions.assertThat(indexes).hasSize(2);
-        Assertions.assertThat(((RangeIndex) indexes.get(0)).getDataType()).isEqualTo(DataType.STRING);
-        Assertions.assertThat(((RangeIndex) indexes.get(0)).getPrecision()).isEqualTo(-1);
-        Assertions.assertThat(((RangeIndex) indexes.get(1)).getDataType()).isEqualTo(DataType.NUMBER);
-        Assertions.assertThat(((RangeIndex) indexes.get(1)).getPrecision()).isEqualTo(-1);
+    public void setAndGetIncludedPath() {
+        String path = "/*";
+        IncludedPath includedPath = new IncludedPath(path);
+
+        List<Index> indexes = new ArrayList<>();
+        indexes.add(Index.range(DataType.STRING, -1));
+        indexes.add(Index.range(DataType.NUMBER, -1));
+        includedPath.setIndexes(indexes);
+
+        List<Index> includedPathIndexes = new ArrayList<>(includedPath.getIndexes());
+        Assertions.assertThat(includedPathIndexes).hasSize(2);
```

Comment:
please have static import so we don't repeat Assertions.

Reply:
added static import